packages/kbn-test/src/es/es_test_cluster.js

```diff
@@ -10,13 +10,12 @@ import elasticsearch from 'elasticsearch';
 
 export function createEsTestCluster(options = {}) {
   const {
-    port = esTestConfig.getPort(),
-    password = 'changeme',
-    license = 'oss',
-    log,
-    basePath = resolve(KIBANA_ROOT, '.es'),
-    // Use source when running on CI
-    from = esTestConfig.getBuildFrom(),
+    port = options.port || esTestConfig.getPort(),
+    password = options.password || 'changeme',
```

Comment:
This was previously using default values for the options...

```js
const { password = options.password || 'changeme' } = options
```which seems really redundant to me